fuzz/fuzz_targets/mutate.rs

```diff
@@ -48,8 +49,22 @@ fuzz_target!(|inputs: (wasm_smith::Module, u64)| {
         }
     };
 
-    let validation_result = wasmparser::validate(&mutated_wasm);
+    // Add valiation with all features enabled, like multiple value return
+    let mut features = WasmFeatures::default();
+    // Add other if needed
+    features.multi_value = true;
+
+    let mut validator = wasmparser::Validator::new();
+    validator.wasm_features(features);
+
+
+    let validation_result = validator.validate_all(&mutated_wasm);
     log::debug!("validation result = {:?}", validation_result);
+
+
+    let mutated_text = wasmprinter::print_bytes(mutated_wasm).unwrap();
+    std::fs::write("mutated.wat", &mutated_text).expect("should write `mutated.wat` okay");
```

Comment:
Can you wrap this in a `if log::log_enabled!(log::Level::Debug) { ... }` like the other places where we write files? This should help keep fuzzer throughput pretty high when we aren't actively debugging a failing input.

Reply:
Done